com.librumreader.librum.yaml

```diff
@@ -0,0 +1,41 @@
+app-id: com.librumreader.librum
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: librum
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --filesystem=host
+  - --device=dri
+modules:
+  - python3-wheel.yaml
+  - python3-setuptools_scm.yaml
+  - python3-scikit-build.yaml
+  - python3-swig.yaml
+  - python3-libclang.yaml
+  - name: librum
+    buildsystem: cmake-ninja
+    build-options:
+      - build-args:
+        - --share-network
```

Comment:
this is not allowed. The build will fail because of that.

Reply:
I got that recommendation from another person, while I was trying to fix my network errors. I will remove it.